datafusion/physical-expr/src/functions.rs

```diff
@@ -1510,44 +1447,6 @@ mod tests {
             Utf8,
             StringArray
         );
-        #[cfg(feature = "crypto_expressions")]
```

Comment:
I double checked and these functions are covered in .slt tests here https://github.com/apache/arrow-datafusion/blob/30d2be9f581846c4b6f875424512eaa7310e16ab/datafusion/sqllogictest/test_files/expr.slt#L1373-L1556

I don't think there is any need to port these rust unit tests too